ledger/accountdb.go

```diff
@@ -551,22 +588,25 @@ func (a *compactResourcesDeltas) resourcesLoadOld(tx *sql.Tx, knownAddresses map
 	var resDataBuf []byte
 	var rtype sql.NullInt64
 	var ok bool
-	for _, entry := range a.misses {
-		idx := a.cache[entry]
-		addr := entry.address
-		delta := a.deltas[idx]
+	for _, missIdx := range a.misses {
+		delta := a.deltas[missIdx]
+		addr := delta.address
 		if delta.oldResource.addrid != 0 {
 			addrid = delta.oldResource.addrid
 		} else if addrid, ok = knownAddresses[addr]; !ok {
 			err = addrRowidStmt.QueryRow(addr[:]).Scan(&addrid)
 			if err != nil {
-				if err == sql.ErrNoRows {
-					err = fmt.Errorf("base account not found while processing resource for addr=%s, aidx=%d", addr.String(), idx)
```

Comment:
does the default error include all info to identify what is missing?

Reply:
ok, added.